clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -1094,6 +1095,8 @@ scheduler_tmpl_t<RecordType, ReaderType>::read_traced_schedule()
     std::vector<std::vector<schedule_output_tracker_t>> all_sched(outputs_.size());
     // Work around i#6107 by tracking counts sorted by timestamp for each input.
     std::vector<std::vector<schedule_input_tracker_t>> input_sched(inputs_.size());
+    std::vector<output_ordinal_t> disk2index;
+    std::vector<uint64_t> disk2cpuid;
     while (options_.replay_as_traced_istream->read(reinterpret_cast<char *>(&entry),
                                                    sizeof(entry))) {
         if (entry.cpu != cur_cpu) {
```

Comment:
Remind the reader that this will happen only at zip file boundaries. #cpu -1 times

Reply:
Done.